mkautodoc/extension.py

```diff
@@ -113,6 +115,25 @@ def trim_docstring(docstring: typing.Optional[str]) -> str:
     while trimmed and not trimmed[0]:
         trimmed.pop(0)
 
+    # replace mentions of built-in types with links to the docs
+    # on docs.python.org
+    for i, x in enumerate(trimmed):
+        for t in [
+            "str", "int", "bytes", "bytearray", "list", "tuple", "range"
+        ]:
+            if t in trimmed[i]:
+                trimmed[i] = trimmed[i].replace(
+                    t,
+                    "".join([
```

Comment:
This could be condensed thoroughly like so:
```py
                trimmed[i] = trimmed[i].replace(
                    t,
                    f"[`{t}`](https://docs.python.org/3/stdtypes.html#{t})"
                )
```

Reply:
Fixed!